### ISSUE TITLE
loss_rs is not used when pretraining ToD-BERT-JNT?

### ISSUE THREAD

**Issue by YawYoung** [jasonwu0731/ToD-BERT#15]
https://github.com/jasonwu0731/ToD-BERT/blob/master/my_tod_pretraining.py#L448

```
## Calculate RCL loss
scores = torch.matmul(hid_cont, hid_resp.transpose(1, 0))
loss_rs = xeloss(scores, resp_label)
loss_rs = loss_rs.item()
loss += loss_rs
```

I think `loss_rs` is not used to update model's parameters.

**Comment by jasonwu0731** [comment]
Hi @YawYoung 

Thanks for pointing this out. This is a mistake while I was refactoring the code and I have corrected it for the latest code. If you check [this previous version](https://github.com/jasonwu0731/ToD-BERT/blob/836f1206f3860f9a5293262bc9e7b6cf21f40704/my_tod_pretraining.py), the code should be correct.

Please make sure while you are training, both losses are reducing. Hope this help.

**Comment by YawYoung** [comment]
Thanks, the loss looks correct now ^-^
